apps/aehttp/priv/swagger.json

```diff
@@ -1224,6 +1224,70 @@
           }
         }
       }
+    },
+    "/contract/compile" : {
+      "post" : {
+        "tags" : [ "external" ],
+        "description" : "Compila a ring contract from source and return byte code",
```

Comment:
"Compila" -> "Compile"

Reply:
Ill fix in my next PR containing tests unless there are other changes.